backend/src/main/kotlin/org/darkSolace/muse/userModule/model/SuspensionHistoryEntry.kt

```diff
@@ -0,0 +1,19 @@
+package org.darkSolace.muse.userModule.model
+
+import java.util.*
+import javax.persistence.*
+
+@Entity
+class SuspensionHistoryEntry(
+    @GeneratedValue
+    @Id
+    val id: Long?,
+    @OneToOne
+    val user: User,
+    @Temporal(TemporalType.TIMESTAMP)
+    val suspendedDate: Date = Date(),
+    val confirmationCode: String = UUID.randomUUID().toString(),
+    @Temporal(TemporalType.TIMESTAMP)
+    var acceptedDate: Date? = null
```

Comment:
Could/should we add confirmationDetails, like browser metadata from the user and OS or something?

Reply:
added a string-field for confirmationDetails. The API endpoint has to changed as well when we know which data and how this data might be sent to the backend.